client/src/routes/Faucet.tsx

```diff
@@ -129,9 +151,23 @@ class Faucet extends PureComponent<{}, FaucetState> {
                                 ) : success ? (
                                     <this.Success />
                                 ) : (
-                                    isLogged && <this.Action />
+                                    isLogged && <this.GetEther />
```

Comment:
```js
<>
  {isLogged && <this.GetEther />}
  {isLogged && showRequestTokens && <this.GetTokens />}
</>                                 
```

Something like this would me more elegant, and would make the duplicated block of code below obsolete

Reply:
fixed